WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -4990,7 +4990,7 @@
 "migration.notifications.actions.secondary.title" = "Decide later";
 
 /* Footer for the migration notifications screen. */
-"migration.notifications.footer" = "When the alert apears tap Allow to continue receiving all your WordPress notifications.";
+"migration.notifications.footer" = "When the alert appears tap Allow to continue receiving all your WordPress notifications.";
```

Comment:
After this PR lands in `release/21.5`, we'll followup with one that lands `release/21.5` into `trunk`. Once that happens, this change will be sent to GlotPress.

I'm not sure how GlotPress will react to it, but my guess is that it will mark all the translations for `migration.notifications.footer` as _fuzzy_ because it should recognize the new value as similar (short [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance)?) as the previous one. In that case, it will be straightforward to approve those strings on our end, without having to wait for community translators to do so.